content/spin/v2/install.md

```diff
@@ -207,13 +207,17 @@ This does not install any Spin templates or plugins. For a starter list, see the
 
 ## Using Cargo to Install Spin
 
-If you have [`cargo`](https://doc.rust-lang.org/cargo/getting-started/installation.html), you can clone the repo and install it to your path:
+If you have [`cargo`](https://doc.rust-lang.org/cargo/getting-started/installation.html), you can clone the repo and install it to your path.
+
+> Note: The `main` branch may have unstable or breaking changes. It is advised to check out the latest Spin release tag, which can be found by navigating to https://github.com/fermyon/spin/releases/latest.
 
 <!-- @selectiveCpy -->
 
 ```bash
-$ git clone https://github.com/fermyon/spin -b v2.0.0
+$ git clone https://github.com/fermyon/spin
 $ cd spin
+$ # Check out the latest tagged release (Linux/Mac)
```

Comment:
This still leaves Windows users with `main` though?

Given the aside you add above, I wonder if changing line 217 to use the fake version number (maybe with a comment like "change this to the latest tag as shown above" or something) is going to end up simpler?

Although I am probably overthinking this.  Such bike, very shed.

Reply:
Ok, updated.  Please take a look when convenient.